### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by weissi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Repository | NIO 2 (Swift 5+) | NIO 1 (Swift 4+)
 [https://github.com/apple/swift-nio-http2][repo-nio-http2]<br> HTTP/2 support | `from: "1.0.0"` | `from: "0.1.0"`
 [https://github.com/apple/swift-nio-extras][repo-nio-extras] <br>useful additions around SwiftNIO | `from: "1.0.0"` | `from: "0.1.0"`
 [https://github.com/apple/swift-nio-transport-services][repo-nio-transport-services] <br> first-class support for macOS, iOS, tvOS, and watchOS | `from: "1.0.0"` | `from: "0.1.0"`
+[https://github.com/apple/swift-ssh][repo-nio-ssh] <br> SSH support | `.upToNextMinor(from: "0.1.0")` | _n/a_
```
Comment (Lukasa) [inline README.md:21]:
```suggestion
[https://github.com/apple/swift-nio-ssh][repo-nio-ssh] <br> SSH support | `.upToNextMinor(from: "0.1.0")` | _n/a_
```

File: README.md [hunk 2]
```diff
@@ -61,7 +62,7 @@ The latest released SwiftNIO 1 version supports Swift 4.0, 4.1, 4.2, and 5.0.
 
 #### SwiftNIO 2
 
-The latest released SwiftNIO 2 version supports only Swift 5.0, 5.1, and 5.2. If you have a SwiftNIO 1 application or library that you would like to migrate to SwiftNIO 2, please check out the [migration guide](docs/migration-guide-NIO1-to-NIO2.md) we prepared for you.
+The latest released SwiftNIO 2 version supports only Swift 5.0, 5.1, 5.2, and 5.3. If you have a SwiftNIO 1 application or library that you would like to migrate to SwiftNIO 2, please check out the [migration guide](docs/migration-guide-NIO1-to-NIO2.md) we prepared for you.
```
Comment (Lukasa) [inline README.md:65]:
```suggestion
The latest released SwiftNIO 2 version supports Swift 5.0, 5.1, 5.2, and 5.3. If you have a SwiftNIO 1 application or library that you would like to migrate to SwiftNIO 2, please check out the [migration guide](docs/migration-guide-NIO1-to-NIO2.md) we prepared for you.
```

We’re at a few years of Swift releases now, we can probably stop saying “only”.

**Review by Lukasa — APPROVED** [review]
Two nits but LGTM

File: README.md (hunk 1 above)
Comment (weissi) [reply README.md:21]: whooops

**Review by shivaylamba — APPROVED** [review]

**Pull request merged**